cypress/support/commands.js

```diff
@@ -12,6 +12,7 @@ import {
   journalSearchMockFile,
 } from './mock_data';
 import { Given, When, Then, And, Before } from 'cypress-cucumber-preprocessor/steps';
+import { dataTestId } from './dataTestids';
```

Comment:
```suggestion
import { dataTestId } from './dataTestIds';
```
siden variabelen får stor "i"

Reply:
hadde glemt at du har sagt man kan importere dette en gang i commands, og bruke det fritt etterpå. må sjekke om det samme virker i frontend også! (lei av å rote fram path til dataTestIds hele tiden)

Reply:
hm, ser ikke ut som det virker like godt i frontend, så lar det ligge der..